sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClientIntegrationTest.java

```diff
@@ -146,157 +145,157 @@ void nonSessionMessageBatch(MessagingEntityType entityType) {
-
-        setSenderAndReceiver(entityType, viaIntermediateEntity, useCredentials);
-
-        final ServiceBusSenderAsyncClient destination1ViaSender = getSenderBuilder(useCredentials, entityType,
-            destinationEntity, false, shareConnection)
-            .viaQueueName(viaQueueName)
-            .buildAsyncClient();
-        final ServiceBusReceiverAsyncClient destination1Receiver = getReceiverBuilder(useCredentials, entityType,
-            destinationEntity, shareConnection)
-            .receiveMode(ReceiveMode.RECEIVE_AND_DELETE)
-            .buildAsyncClient();
-
-        final AtomicReference<ServiceBusTransactionContext> transaction = new AtomicReference<>();
-
-        // Act
-        try {
-            StepVerifier.create(destination1ViaSender.createTransaction())
-                .assertNext(transactionContext -> {
-                    transaction.set(transactionContext);
-                    assertNotNull(transaction);
-                })
-                .verifyComplete();
-            assertNotNull(transaction.get());
-
-            StepVerifier.create(sender.sendMessages(messages, transaction.get()))
-                .verifyComplete();
-            StepVerifier.create(destination1ViaSender.sendMessages(messages, transaction.get()))
-                .verifyComplete();
-            StepVerifier.create(destination1ViaSender.sendMessages(messages, transaction.get()))
-                .verifyComplete();
-
-            StepVerifier.create(destination1ViaSender.commitTransaction(transaction.get())
-                .delaySubscription(Duration.ofSeconds(1)))
-                .verifyComplete();
-
-            // Assert
-            // Verify message is received by final destination Entity
-            StepVerifier.create(destination1Receiver.receiveMessages().take(totalToDestination).timeout(shortTimeout))
-                .assertNext(receivedMessage -> {
-                    assertMessageEquals(receivedMessage, messageId, isSessionEnabled);
-                    messagesPending.decrementAndGet();
-                })
-                .assertNext(receivedMessage -> {
-                    assertMessageEquals(receivedMessage, messageId, isSessionEnabled);
-                    messagesPending.decrementAndGet();
-                })
-                .verifyComplete();
-
-            // Verify, intermediate-via queue has it delivered to intermediate Entity.
-            StepVerifier.create(receiver.receiveMessages().take(total).timeout(shortTimeout))
-                .assertNext(receivedMessage -> {
-                    assertMessageEquals(receivedMessage, messageId, isSessionEnabled);
-                    messagesPending.decrementAndGet();
-                })
-                .verifyComplete();
-        } finally {
-            destination1Receiver.close();
-            destination1ViaSender.close();
-        }
-    }
+    // @Test
+    // void viaQueueMessageSendTest() {
```

Comment:
Should we remove them instead of comment out?

Reply:
I kept them so that we can use them in the next update when we add the transaction feature for cross entities back in

Reply:
I can delete the entire file if we feel otherwise.

@conniey, @hemanttanwar ?

Reply:
Normally we delete the test if we do not need it soon. We can always get it back from git history and old commits.
If we are going to introduce this next months or soon, we can disable also add `@Disable` annotation.

`@Disabled("Reason why disable.")`

There are other test in this file, so we can not delete the file but remove the test.